src/child/constants/actionTypes.js

```diff
@@ -22,5 +22,8 @@ export const WINDOW = {
     STATE_EXPANDED: 'STATE_EXPANDED',
     STATE_FULL_VIEW: 'STATE_FULL_VIEW',
     RESIZING: 'RESIZING',
-    RESIZE_ERROR: 'RESIZE_ERROR'
+    RESIZE_ERROR: 'RESIZE_ERROR',
+    UPDATING_OPTIONS: 'UPDATING_OPTIONS',
+    OPTIONS_SUCCESS: 'OPTIONS_SUCCESS',
+    OPTIONS_ERROR: 'OPTIONS_ERROR'
```

Comment:
I'd add the verb to these actions - `UPDATING_OPTIONS_SUCCESS`, `UPDATING_OPTIONS_ERROR`